src/frontend/pages/login/login.js

```diff
@@ -21,12 +21,11 @@ function Login() {
         type: "TOKEN",
         payload: response.data.encodedToken,
       });
-      navigate("/");
+      navigate("/home");
     } catch (err) {
       authDispatch({
```

Comment:
please tell me regarding this auth dispath

Reply:
come to VC